pandas/core/apply.py

```diff
@@ -307,23 +305,37 @@ def apply_standard(self):
                 # reached via numexpr; fall back to python implementation
                 pass
             else:
-                return self.obj._constructor_sliced(result, index=labels)
+                # this means that the reduction was successful
+                if partial_result is None:
```

Comment:
this seems excessively complex

Reply:
What would be a better solution?
returning the partial result as the first result in the returned array, and returning a boolean value for reduction success?

Reply:
I agree this is also too complex. Not sure of the solution; what you suggest is probably better so I think worth looking at that instead as a patch and can iterate on that

Reply:
I changed it per my proposed solution.
Please let me know if its simple enough